packages/modul-components/src/components/menu/menu-item/menu-item.html

```diff
@@ -1,13 +1,13 @@
 <li class="m-menu-item"
     :class="{'m--is-selected': selected,
-             'm--is-group-selected': groupSelected,
              'm--is-disabled': isDisabled,
              'm--is-group': group,
              'm--is-open': propOpen,
-             'm--is-inside-group': insideGroup,
-             'm--is-anim-ready': isAnimReady}"
```

Comment:
Conserver la class **m--is-anim-ready**. Elle sert à arrêter la transition des éléments lorsque composant n'a pas terminé son cycle de vie.

Reply:
Est-ce que ça l'a vraiment une utilité ? Ça créait pleins de glytches, et de flickers avec les transition de hover entre autre...